### ISSUE TITLE
See if I can use dconf to allow users to set preferred terminal 

### ISSUE THREAD

**Issue by Dvlv** [Dvlv/BoxBuddyRS#47]
Only worth doing if it would work in the flatpak too

**Comment by Dvlv** [comment]
```
$ dconf write /org/gnome/boxbuddy/default-terminal "'kgx'"

$ dconf read /org/gnome/boxbuddy/default-terminal
'kgx'
```

**Comment by Dvlv** [comment]
In progress, using proper gio::Settings

**Comment by Dvlv** [comment]
Done in Master

**Comment by Dvlv** [comment]
TODO: Docs

**Comment by Dvlv** [comment]
Released in 2.1.0
